style.css

```diff
@@ -1,3 +1,3 @@
 .text {
-  color: yellow;
+  color: red;
```

Comment:
- [ ] This is an inline comment and it is required you do at least one. This will help you as a developer in giving reviews.

Reply:
i don't understand your comment

Reply:
I mean you should comment on a code, at least one comment.

Reply:
Dude, are you there? Can we discuss this on Slack? I am reviewing your re-review at the moment.